homepage.php

```diff
@@ -52,57 +52,57 @@
 				<div class="overlay">
-    				<div class="text"><a href = "Query8.php">Indie</a></div>
+    				<div class="text"><a href = "Query8_admin.php">Indie</a></div>
   				</div>
 			</div>
 			<div class="images2">
 				<a href = "Query7.php"><img src = "images/R&B.jpg" class="image2"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query7.php">R and B</a></div>
+    				<div class="text"><a href = "Query7_admin.php">R and B</a></div>
   				</div>
 			</div>
 			<div class="images3">
 				<a href = "Query6.php"><img src = "images/Pop.jpg" class="image3"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query6.php">Pop</a></div>
+    				<div class="text"><a href = "Query6_admin.php">Pop</a></div>
   				</div>
 			</div>
 			<div class="images4">
 				<a href = "Query10.php"><img src = "images/Rock.jpg" class="image4"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query10.php">Rock</a></div>
+    				<div class="text"><a href = "Query10_admin.php">Rock</a></div>
   				</div>
 			</div>
 			<Space></Space>
 			<div class="images5">
 				<a href = "Query9.php"><img src = "images/Easy Listen.jpg" class="image5"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query9.php">Easy Listen</a></div>
+    				<div class="text"><a href = "Query9_admin.php">Easy Listen</a></div>
   				</div>
 			</div>
 			<div class="images6">
 				<a href = "Query3.php"><img src = "images/Country.jpg" class="image6"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query3.php">Country</a></div>
+    				<div class="text"><a href = "Query3_admin.php">Country</a></div>
   				</div>
 			</div>
 			<div class="images7">
 				<a href = "Query4.php"><img src = "images/Jazz.jpg" class="image7"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query4.php">Jazz</a></div>
+    				<div class="text"><a href = "Query4_admin.php">Jazz</a></div>
   				</div>
 			</div>
 			<div class="images8">
 				<a href = "Query5.php"><img src = "images/Dance.jpg" class="image8"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query5.php">Dance</a></div>
+    				<div class="text"><a href = "Query5_admin">Dance</a></div>
```

Comment:
2/9/2021: So I make a page called Query5_admin will mostly the same with Query5 but the only difference was in Query5_admin I use the admin nav which can access to the setting. So for admin page I should make it to link to Query5_admin.